app/src/main/java/com/futurice/freesound/viewmodel/Binder.java

```diff
@@ -16,11 +16,13 @@
 
 package com.futurice.freesound.viewmodel;
 
+import android.support.annotation.NonNull;
+
 import io.reactivex.disposables.CompositeDisposable;
 
 public interface Binder {
 
-    void bind(final CompositeDisposable disposables);
+    void bind(@NonNull final CompositeDisposable disposables);
```

Comment:
Unlike the `@NonNull`, the `final` won't be enforced from the interface could you remove it.